lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -77,9 +77,11 @@ class AxeAudit extends Audit {
       items = rule.nodes.map(node => ({
         node: /** @type {LH.Audit.Details.NodeValue} */ ({
           type: 'node',
+          // This selector represents the aXe css selectors
+          // Node selector can be accessed with node.selector
```

Comment:
```suggestion
          // This selector represents the aXe CSS selectors
```

Reply:
Since we're ignoring aXe selector in this case, the comment is not necessary anymore.